src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cleanSource = sourceString.trim();
```

Comment:
[CODE KNOWLEDGE] - Don't need to make copy before each operation. Moreover, methods like `filter` or `map` don't mutate original array. In this case, the `trim` method does not mutate the original string, and it's unnecessary to store the result in `cleanSource` since you're not using the original `sourceString` anywhere else.